### ISSUE TITLE
Built in screen reader in MS Edge does not read Input-fields

### ISSUE THREAD

**Issue by xmrsa** [Altinn/app-frontend-react#1622]
### Description of the bug

The screen reader in Edge consistently skips the **Input-fields** in an app, that is fields with numbers, short text or date fields. It actually manages to read the content of the Text Area fields. 

### Steps To Reproduce

1. Open an app with an **input** field in Microsoft Edge
2. Hit **Ctrl + Shift + U** to start the screen reader
3. Pay attention as it skips the content of all short Input-field.
